core/primitives/src/version.rs

```diff
@@ -176,15 +176,14 @@ pub const PROTOCOL_VERSION: ProtocolVersion = if cfg!(feature = "nightly_protoco
 
 /// The points in time after which the voting for the latest protocol version
 /// should start.
-pub static PROTOCOL_UPGRADE_SCHEDULE: Lazy<Option<ProtocolUpgradeVotingSchedule>> =
-    Lazy::new(|| {
-        if cfg!(feature = "shardnet") {
-            Some(ProtocolUpgradeVotingSchedule::from_str("2022-09-05 15:00:00").unwrap())
-        } else {
-            // Update to latest protocol version on release.
-            None
-        }
-    });
+pub const PROTOCOL_UPGRADE_SCHEDULE: Lazy<ProtocolUpgradeVotingSchedule> = Lazy::new(|| {
+    if cfg!(feature = "shardnet") {
+        ProtocolUpgradeVotingSchedule::from_str("2022-09-05 15:00:00").unwrap()
+    } else {
+        // Update to latest protocol version on release.
+        ProtocolUpgradeVotingSchedule::default()
```

Comment:
Can we use the time from the last upgrade here? So this doesn't flip between default and non-default, and just always points to when it happened the last time?

Reply:
That will never stay in sync.  This is only ever updated on release branch and never on master.

Reply:
Ah, makes sense. Could you maybe add that to the comment?